.yarn/versions/69ddbe20.yml

```diff
@@ -0,0 +1,30 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/core": patch
+  "@yarnpkg/plugin-compat": patch
+  "@yarnpkg/plugin-npm": patch
+  "@yarnpkg/plugin-npm-cli": patch
```

Comment:
Should only need to release `@yarnpkg/plugin-npm` and `@yarnpkg/cli` for this

Reply:
@merceyz not familiar with this part, it has been changed now.